prisma/mongoSchema.prisma

```diff
@@ -35,9 +34,14 @@ enum Status {
   EDITING_COMPLETE
 }
 
+enum AuditType {
+  WORKFLOW
```

Comment:
consider renaming to WORKFLOW_ASSIGNMENT since that is the only scenario under which this audit will be recorded